ansible/inventories/devnet-0/group_vars/ethereumjs.yaml

```diff
@@ -29,4 +29,4 @@ ethereumjs_container_command_extra_args:
   - --maxPeers=75
   - --isSingleNode=true
   - --logLevel=debug
-  #- --trustedSetup=/network-config/trusted_setup.txt
\ No newline at end of file
+  - --trustedSetup=/network-config/trusted_setup.txt
```

Comment:
would leave trusted_setup related configs out tbh. its a devnet specific thing and not really relevant or needed to change for anything other than dencun testing

Reply:
also there's a chance not all clients support the flag on their mainline releases yet, which is what we are targeting and leaving it out would ensure the most standard behaviour

Reply:
removed